example/src/main/AndroidManifest.xml

```diff
@@ -10,7 +10,7 @@
             android:supportsRtl="true"
             android:theme="@style/AppTheme">
 
-        <activity android:name="org.mozilla.fxa_data.example.FirefoxDataInRecyclerViewExampleActivity">
+        <activity android:name="org.mozilla.fxa_data.example.KotlinExampleActivity">
```

Comment:
Maybe `FirefoxDataInRecyclerViewExampleKotlinActivity`? I'd prefer if it was clear that this is:
1. Puts the Data in the RecyclerView
2. Is a port of the other activity.

Reply:
Good idea!